site/_posts/2017-02-10-love-is-in-the-PRs.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Last Week in ManageIQ: Love is in the PRs"
+author: Jillian Tullo
```

Comment:
Might be better to use jntullo here, as you did for your previous 2 posts. In addition I think there are plans to link author ids to github accounts. (ping @hayesr, any ETA for this?)